frontend/src/pages/ErrorsV2/SearchPanel/SearchPanel.tsx

```diff
@@ -133,13 +143,32 @@ const SearchPanel = () => {
 					],
 				},
 			}
+			const clickhouseQuery: ClickhouseQuery = JSON.parse(searchQuery)
+			const newRules = clickhouseQuery.rules.filter(
+				(r) => r[0] !== 'error-field_timestamp',
+			)
+			newRules.push([
+				'error-field_timestamp',
+				'between_date',
+				new Date(Date.parse(lte)).toISOString() +
+					'_' +
+					new Date('2050-01-01').toISOString(),
```

Comment:
I'm assuming this we always want this in the future? Should we do something like:
`new Date(Date.now() +  7 * 24 * 60 * 60 * 1000).toISOString()` to make sure it will be in the future, even when 2050 comes 😅